### DISCUSSION TITLE
The use of WebActions

### DISCUSSION THREAD

**Discussion by cc359** [akshayp7/playwright-typescript-playwright-test#31] [General]
Hi!

To start off, this is a great project to draw some inspiration from, great work!

I'm just wondering why the decision has been made to use WebActions to interact with the SUT instead of using the actions available from Playwright, seeing as this is a Playwright based framework?

Thanks for your time!

**Comment by akshayp7** [comment]
Thanks for feedback. I used we actions class because for example if you want to use drag and drop in multiple places instead of calling drag and the. Drop we can directly use drag drop single method to perform actions. That was my understanding when I was designing the project to segregate actions at one place.
